File: MasterMindJS/masterMindLogic.js
class masterMindLogic {
    static receiveCode(level) {
        var code = "";
        for (let i = 0; i < 4; i++) {
            code += String.fromCharCode('A'.charCodeAt(0) + Math.random()*(level + 2));
        }
        return code;
    }
    

    static GetGuessFeedback(code, input) {
        return "GetGuessFeedback not implemented yet. Try to apply TDD practices and implement it now!";
    }
}

module.exports = masterMindLogic;

File: MasterMindPython/masterMindLogic.py
import random

class MasterMindLogic:
    @staticmethod
    def receive_code(level):
        code = ""
        for _ in range(4):
            code += chr(ord('A') + random.randint(0, level + 1))
        return code

    @staticmethod
    def get_guess_feedback(code, input_code):
        return "GetGuessFeedback not implemented yet. Try to apply TDD practices and implement it now!"
